metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -872,7 +873,15 @@ def _generate_single_profile(
                     ge_config["schema"] = temp_table_db
 
             if self.config.bigquery_temp_table_schema:
-                bigquery_temp_table = f"{temp_table_db}.{self.config.bigquery_temp_table_schema}.ge-temp-{uuid.uuid4()}"
+                num_parts = self.config.bigquery_temp_table_schema.split(".")
+                if len(num_parts) == 1:
```

Comment:
Can you add comments on the PR for the special cases added? It is not clear why we have these special cases